### ISSUE TITLE
Child job PR status updates broken in 1.36.0+

### ISSUE THREAD

**Issue by mshuler** [jenkinsci/ghprb-plugin#506]
Downgrading to 1.35.0 is the only fix that got child job status updates to pull requests working again. No job configurations have changed for many moons. 

I upgraded to 1.36.0 on Apr 10 and downstream ghprb jobs failed to send any status to the PRs. I upgraded to 1.36.1 without that fixing anything, but I thought it may, since I generally use all the default messages and leave the configurations mostly blank, with the exception of a context line.

1.35.0 sample console:
```
19:32:53 Started by upstream project "cassandra-pr-artifacts" build number 596
19:32:53 originally caused by:
19:32:53  GitHub pull request #160 of commit 20a7a24a0ea806316df988d14e7c9c2792dea017, no merge conflicts.
19:32:53 Setting status of 20a7a24a0ea806316df988d14e7c9c2792dea017 to PENDING with url http://jenkins-cassandra.datastax.lan/job/cassandra-pr-dtest/523/ and message: 'Build started'
19:32:53 Using context: jenkins/dtest
19:32:53 [EnvInject] - Loading node environment variables.
19:32:54 Building on master in workspace /var/lib/jenkins/jobs/cassandra-pr-dtest/workspace
19:32:54 Wiping out workspace first.
...
```

1.36.0/1 sample console:
```
17:12:36 Started by upstream project "cassandra-pr-artifacts" build number 610
17:12:36 originally caused by:
17:12:36  GitHub pull request #161 of commit 3d938418f13e1052a091b362371b1ad3e2aa2248, no merge conflicts.
17:12:36 [EnvInject] - Loading node environment variables.
17:12:36 Building on master in workspace /var/lib/jenkins/jobs/cassandra-pr-dtest/workspace
17:12:36 Wiping out workspace first.
...
```

Downgrading back to 1.35.0 has PRs getting status updates from our child jobs as expected.

**Comment by benpatterson** [comment]
@mshuler could you describe your configuration a bit more? How is your downstream job called, and how are its parameters configured, for example?

**Comment by mshuler** [comment]
Upon success, the job cassandra-pr-artifacts (ant build) starts up 2 different test jobs, cassandra-pr-test and cassandra-pr-dtest, as "Trigger parameterized build on other projects" and sends "Current build parameters". Those 2 test jobs were picking up all the ghprb env vars and running fine, just never pinging Github when triggered or completed.

Those console snippets are from one of the downstream triggered jobs, one from a few days prior, running 1.35.0, and one after upgrading to 1.36.0. Nothing was changed in the configurations, and they've been running OK for months. I sure thought perhaps I needed to update some new configuration option, but couldn't locate what that might be, and a downgrade "just worked".

System configuration wise, we do run Jenkins with `-Dhudson.model.ParametersAction.keepUndefinedParameters=true` in order to pass all the parameters through. I don't recall when that security bit was added to Jenkins, but it's easier than specifying every one allowed and we're in a private lab anyway.

The only context we set is "jenkins" in the main Jenkins configuration, "jenkins/artifacts" on the artifacts job, "jenkins/test" on -test, and "jenkins/dtest" on the -dtest job. No other status strings or anything are configured, so they are just the defaults. All I really care about is being able to tell which link was which job in the Github PR, since there are (should be :) ) 3 links on our pull requests.

Let me know if I can give any additional info!
